tests/e2e/specs/Modeler.spec.js

```diff
@@ -568,4 +568,16 @@ describe('Modeler', () => {
           });
       });
   });
+
+  it('after collapsing panels, show inspector handel when element is highlighted', () => {
```

Comment:
```suggestion
  it('after collapsing panels, show inspector handle when element is highlighted', () => {
```

Reply:
Whoops, should be "inspector panel". Good catch!